python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -120,11 +128,15 @@ def add_gpu_object(
         Args:
             obj_id: The object ID of the GPU object.
             gpu_object: A list of tensors representing the GPU object.
+            num_readers: The number of readers for the GPU object. It should only
+              be set for receiver actors.
```

Comment:
Docs descriptions survive better if we just say what the logic in this function does instead of saying how the caller uses it. E.g., we should just say what gpu_object_store will clean up the object after num_readers calls to some function, instead of saying anything about the actors that will call it.